boards/arm/nucleo_wba52cg/doc/nucleo_wba52cg.rst

```diff
@@ -220,12 +220,31 @@ Default settings are 115200 8N1.
 Programming and Debugging
 *************************
 
+Nucleo WBA52CG board includes an ST-LINK/V3 embedded debug tool interface.
+It could be used for flash and debug using either OpenOCD or STM32Cube ecosystem tools.
+
+OpenOCD Support
+===============
+
+For now, openocd support is available only on `STMicroelectronics OpenOCD fork`_.
+In order to use it, you should clone and compile it following usual README
+guidelines.
+Once it is done, you can set the OPENOCD and OPENOCD_DEFAULT_PATH variables in
+:zephyr_file:`boards/arm/nucleo_wba52cg/board.cmake` to point the build
+to the paths of the OpenOCD binary and its scripts,  before
+including the common openocd.board.cmake file:
+
+   .. code-block:: none
+
+      set(OPENOCD "<path_to_opneocd_repo>/src/openocd" CACHE FILEPATH "" FORCE)
```

Comment:
remove filename between double quotes. it does matter but that's the syntax used in all the other .cmake files I think?